### DISCUSSION TITLE
[iOS] sync to firefox?

### DISCUSSION THREAD

**Discussion by sothawo** [floccusaddon/floccus#1583] [Q&A] [answered]
Does the iOS app just disply the synced bookmarks, os does it integrate with firefox on iOS, so that the bookmarks are available in firefox itself?

**Comment by marcelklehr** [comment, marked as answer]
Hi @sothawo 
Both mobile apps do *not* integrate with the browser apps you have installed on your phone, but are *standalone* apps. Firefox for Android has announced support for web extensions dealing with bookmarks in the upcoming months, however; for Firefox iOS as well as other iOS browsers I don't know anything
